modules/cluster/storage.tf

```diff
@@ -9,6 +9,7 @@ resource "aws_s3_bucket" "logs_jenkins_x" {
   tags = {
     Owner = "Jenkins-x"
   }
+  force_destroy = true
```

Comment:
Actually, I am inclined to align this with what we do in the [terraform-google-jx](https://github.com/jenkins-x/terraform-google-jx/blob/master/modules/cluster/storage.tf#L13). There we use a variable which allows you to set the _force_destroy_ value. The default is in this case defensive set to _false_ to avoid accidental data loss.

This is paired with a note in the README:

> If you just want to experiment with Jenkins X, you can set force_destroy to true. This allows you to remove all generated resources when running terraform destroy, including any generated buckets including their content.

@ankitm123 - what do think?

Reply:
Agreed @hferentschik. A variable makes sense. I will update the PR.

Reply:
In that case, we should also mention in the docs, how they should delete it, as terraform destroy would fail if they try to delete the buckets with force_destroy set to false (they have to empty it, and then re-run terraform destroy).

Reply:
> In that case, we should also mention in the docs, how they should delete it, as terraform destroy would fail if they try to delete the buckets with force_destroy set to false (they have to empty it, and then re-run terraform destroy

+1 for being more explicit in the docs as well.